src/samples/Uno.Themes.Samples/Uno.Themes.Samples.Shared/Content/Controls/TimePickerSamplePage.xaml

```diff
@@ -40,7 +40,6 @@
 
 							<TimePicker Header="Select a Time"
 										Style="{StaticResource MaterialTimePickerStyle}"
-										ios:FlyoutPresenterStyle="{StaticResource MaterialTimePickerFlyoutPresenterStyle}"
```

Comment:
Oh, you already commented on that in https://github.com/unoplatform/Uno.Themes/pull/423#discussion_r493676442 @Xiaoy312 ! The reason this was needed is the bug in `TimePicker.xaml` and XAML generator didn't produce an error since `ShouldWriteErrorOnInvalidXaml` wasn't set to true. Your comment was missed at that time and the PR got merged without addressing the comment.

Reply:
nice archaeology